README.md

```diff
@@ -1,4 +1,5 @@
 # Graph Object Mapper (Grom)
```

Comment:
We need to write a new Readme that reflects the changes we have made - this should be done at a later date